src/silo/storage/sequence_store.test.cpp

```diff
@@ -0,0 +1,47 @@
+#include "silo/storage/sequence_store.h"
+
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <string>
+
+std::unique_ptr<silo::SequenceStore> setupSequenceStore() {
+   std::unique_ptr<silo::SequenceStore> sequence_store = std::make_unique<silo::SequenceStore>();
+
+   // Construct test genomes.
+   std::string test_genome1 = "A";
+   test_genome1.reserve(silo::GENOME_LENGTH);
+   for (unsigned i = 1; i < silo::GENOME_LENGTH; ++i) {
+      test_genome1.push_back('A');
+   }
+
+   std::string test_genome2 = "C";
+   test_genome2.reserve(silo::GENOME_LENGTH);
+   for (unsigned i = 1; i < silo::GENOME_LENGTH; ++i) {
+      test_genome2.push_back('C');
+   }
+
+   sequence_store->interpret({std::move(test_genome1), std::move(test_genome2)});
+
+   return sequence_store;
+}
+
+TEST(SequenceStore, shouldReturnCorrectBitmapForAmbiguousSymbol) {
+  auto sequence_store = setupSequenceStore();
+
+  // Note: 1-indexed!
```

Comment:
Is this comment necessary, or does it belong to #61 ?

Reply:
Well, this comment was rather for myself because I didn't know in the beginning that genomes are 1-indexed 😅